composer.json

```diff
@@ -18,22 +18,16 @@
             "ZendServiceTest\\ReCaptcha\\": "test/"
         }
     },
-    "repositories": [
-        {
-            "type": "composer",
-            "url": "http://packages.zendframework.com/"
-        }
-    ],
     "require": {
-        "php": ">=5.5",
-        "zendframework/zend-config": "~2.0",
-        "zendframework/zend-http": "~2.0",
-        "zendframework/zend-json": "~2.0"
+        "php": "^5.6 || ^7.0",
+        "zendframework/zend-http": "^2.5.4",
+        "zendframework/zend-json": "^2.6.1 || ^3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.0",
+        "phpunit/phpunit": "^5.7 || ^6.0",
         "squizlabs/php_codesniffer": "^2.3",
```

Comment:
Change this dependency to `"zendframework/zend-coding-standard": "~1.0.0"`, please, and then re-run phpcs and/or phpcbf; they'll pick up the various CS bits I've noticed.